api/src/mailbox.rs

```diff
@@ -976,6 +983,40 @@ impl Default for SetAuthManifestReq {
     }
 }
 
+// GET_IDEVID_CSR
+#[repr(C)]
+#[derive(Default, Debug, AsBytes, FromBytes, PartialEq, Eq)]
+pub struct GetIDevIDCSRReq {
```

Comment:
minor nit: The other APIs don't capitalize the acronyms. e.g. `GetIdevIdCsrReq`

Reply:
Done. I'll wait to push these changes until the UART logs PR is merged.